File: trunk/Source/Assets/Scripts/camera/SimpleAnim.js
#pragma strict

public var RowStart 			: int = 0;
public var ColumnStart 			: int = 0;
public var TotalFrames 			: int = 1;
public var Orientation 			: int = 1;
//public var FramesPerSeconds 	: int = 12;


private var SimpleAnim 		: AnimSprite;  



function Start () {

	SimpleAnim = GetComponent("AnimSprite")as AnimSprite;

	while(true) 
		yield CoUpdate();
}

function CoUpdate ()
{
	
	SimpleAnim.PlayFrames( RowStart, ColumnStart, TotalFrames, Orientation);

	yield;

}

File: trunk/Source/Assets/Scripts/custom/SimpleAnim.cs
using UnityEngine;
using System.Collections;

public class SimpleAnim : MonoBehaviour
{
    public int RowStart = 0;
    public int ColumnStart = 0;
    public int TotalFrames = 1;
    public int Orientation = 1;
    //public int FramesPerSeconds = 12;

    private AnimSprite simpleAnim;

    void Start(){

        simpleAnim = gameObject.GetComponent<AnimSprite>()as AnimSprite;

        //StartCoroutine(CoUpdate());                                 // Changed Because it can't hold re-activation
}
    void Update()
    {
            simpleAnim.PlayFrames(RowStart, ColumnStart, TotalFrames, Orientation);
    }

    //IEnumerator CoUpdate()
    //{
    //    while (true)
    //    {
    //        simpleAnim.PlayFrames(RowStart, ColumnStart, TotalFrames, Orientation);
    //        yield return 0;
    //    }
    //}
}